subgraph-kovan-staging.yaml

```diff
@@ -107,23 +107,128 @@ dataSources:
         - function: update(bytes32,uint256)
           handler: handleNftFeedUpdate
       file: ./src/mapping.ts
+  # - kind: ethereum/contract
+  #   name: SeniorTranche
+  #   network: kovan
+  #   source:
+  #     address: "0xA5d9B0190874b29d824B0d0164dec8B6034d1615"
+  #     abi: SeniorTranche
+  #     startBlock: 18594066
+  #   mapping:
+  #     kind: ethereum/events
+  #     apiVersion: 0.0.3
+  #     language: wasm/assemblyscript
+  #     entities:
+  #       - Pool
+  #     abis:
+  #       - name: SeniorTranche
+  #         file: ./abis/SeniorTranche.json
+  #     callHandlers:
+  #       - function: file(bytes32,uint256)
+  #         handler: handleSeniorTrancheFile
+  #     file: ./src/mapping.ts
   - kind: ethereum/contract
-    name: SeniorTranche
+    name: Block2
```

Comment:
We only need 1 block handler, since there is only one block for all deployments (they are all on the same chain). Can we remove this one?

Reply:
sure